kolibri/plugins/pdf_viewer/assets/src/views/PdfRendererIndex.vue

```diff
@@ -378,4 +400,31 @@
     }
   }
 
+  .fullscreen-button {
+    margin: 0;
+
+    svg {
+      position: relative;
+      top: 8px;
+    }
+  }
+
+  .fullscreen-header {
+    height: $controls-height;
+    text-align: end;
+  }
+
+  .slide-enter-active {
```

Comment:
Let's use the same material design easing functions as the scrolling header e.g.

https://github.com/learningequality/kolibri/blob/0498cfd39a6c1d70e5646470a91b515619d03cd2/kolibri/core/assets/src/views/CoreBase/ScrollingHeader.vue#L150-L160

Reply:
These need to be imported from the design system `@import '~kolibri-design-system/lib/styles/definitions';`